WebDevStudios/wd_s
# Moves package dependencies

Closes #563

### DESCRIPTION

Moves some required dependencies to `dependencies` and keeps the rest in `devDependencies` in the package file.

### SCREENSHOTS

![screenshot of dependencies and devDependencies](https://d.pr/i/UrTT2V+)

### OTHER


### STEPS TO VERIFY

1. Check out the branch
2. Run `npm install --production && npm run build`
3. Load the site and verify everything is built as expected

### DOCUMENTATION

I don't believe we'll need to update any documentation here.
